cmd/skyeye/main.go

```diff
@@ -81,6 +82,8 @@ func init() {
 	_ = skyeye.MarkFlagRequired("whisper-model")
 	voiceFlag := NewEnum(&voiceName, "Voice", "", "feminine", "masculine")
 	skyeye.Flags().Var(voiceFlag, "voice", "Voice to use for SRS transmissions (feminine, masculine)")
+	playbackSpeedFlag := NewEnum(&playbackSpeed, "string", "veryslow", "slow", "standard", "fast", "veryfast")
```

Comment:
The first enum value is used as a default, so "standard" should be before the others

Reply:
didnt know this thx. Fixed it

Reply:
Yeah that enum type is my own crap, not part if cobra, lol